tests/test_validations.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from hypothesis import given, strategies as st
+
+from mawpy.utilities.validations import check_non_negative
+
+logger = logging.getLogger(__name__)
+
+
+@given(st.floats(min_value=0, allow_infinity=False, allow_nan=False))
+def test_valid_non_negative_values(value):
+    # Hypothesis generates various non-negative floats, including edge cases like 0.0
+    check_non_negative(value)
+    pass
```

Comment:
I see 2 tests for the `check_non_negative` please add test for `validate_input_args` as well.

Reply:
added